pkg/rhcos/builds.go

```diff
@@ -23,7 +25,8 @@ type metadata struct {
 	OSTreeVersion string `json:"ostree-version"`
 }
 
-func fetchRHCOSBuild(ctx context.Context) (*metadata, error) {
+// FetchRHCOSBuild retrieves the pinned RHEL CoreOS version.
+func FetchRHCOSBuild(ctx context.Context) (*Metadata, error) {
```

Comment:
Well the documentation on this function is incorrect, it is not only returning the version but, also returns all the other things.